airbyte-webapp/src/components/ui/TagInput/TagInput.tsx

```diff
@@ -20,17 +19,29 @@ const customStyles = {
     borderRadius: `${styles.borderRadius}`,
     paddingLeft: `${styles.paddingLeft}`,
   }),
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any -- same as above
-  multiValueLabel: (provided: any) => ({
+  multiValueLabel: (provided) => ({
     ...provided,
     color: `${styles.fontColor}`,
     fontWeight: 500,
   }),
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any -- same as above
-  multiValueRemove: (provided: any) => ({
+  multiValueRemove: (provided) => ({
     ...provided,
     borderRadius: `${styles.borderRadius}`,
   }),
+  control: (provided, state) => {
```

Comment:
One thing I noticed when testing locally was that when I hovered my cursor over the TagInput it stays as the `default` cursor, whereas when I hover over the other text input fields it changes to a `text` cursor. Could we also make this TagInput have a `text` cursor, so that it is consistent? (And I feel a text cursor also makes sense here because the user is still expected to type values into the field)

Reply:
good catch, fixed

Reply:
@flash1293 one more small nit here: when hovering over the `X` close button of one of the tags, could we go back to the `default` cursor (or maybe `pointer`, since that seems to be what we use for buttons)? It currently uses the text cursor which looks a little odd.

We should also use the same pointer style for the whole-component clear `X` button on the right side